lib/config/admin.ts

```diff
@@ -1,30 +1,34 @@
 import type { RenovateConfig, RepoAdminConfig } from './types';
 
-let adminConfig: RepoAdminConfig = {};
+let adminConfig: RepoAdminConfig;
 
 // TODO: once admin config work is complete, add a test to make sure this list includes all options with admin=true (#9603)
-export const repoAdminOptions = [
-  'allowCustomCrateRegistries',
-  'allowPostUpgradeCommandTemplating',
-  'allowScripts',
-  'allowedPostUpgradeCommands',
-  'customEnvVariables',
-  'dockerChildPrefix',
-  'dockerImagePrefix',
-  'dockerUser',
-  'dryRun',
-  'exposeAllEnv',
-  'privateKey',
-];
+const adminDefaults: RepoAdminConfig = {
+  allowCustomCrateRegistries: undefined,
+  allowPostUpgradeCommandTemplating: undefined,
+  allowScripts: undefined,
+  allowedPostUpgradeCommands: undefined,
+  customEnvVariables: undefined,
+  dockerChildPrefix: undefined,
+  dockerImagePrefix: undefined,
+  dockerUser: undefined,
+  dryRun: undefined,
+  exposeAllEnv: undefined,
+  privateKey: undefined,
+  localDir: '',
+  cacheDir: '',
```

Comment:
What are the implications for these being empty strings? Makes some tests shorter because they don't need to populate the fields?

Reply:
I think I can return it to previous state. It was useful at the first stage for not to crash lots of existing working tests.